src/core/public/doc_links/doc_links_service.ts

```diff
@@ -167,6 +167,7 @@ export class DocLinksService {
           timelionDeprecation: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/dashboard.html#timelion-deprecation`,
           lens: `${ELASTIC_WEBSITE_URL}what-is/kibana-lens`,
           maps: `${ELASTIC_WEBSITE_URL}maps`,
+          vega: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/vega-graph.html`,
```

Comment:
This URL doesn't exist any more. Try https://www.elastic.co/guide/en/kibana/7.10/vega-reference.html instead

Reply:
Yep, that's the point of (the first commit in) this PR. I wanted to make sure the docs build failed, before I replaced it with the correct URL. 

Unfortunately, the build passed 😞 . Looks like I have some more work to do on the docs side.

Reply:
Should have read it more closely!

Reply:
It looks like, starting in 7.11, the page is now called just `vega.html`.